example/CMakeLists.txt

```diff
@@ -31,6 +31,6 @@ add_executable(osrm-example example.cpp)
 
 find_package(LibOSRM REQUIRED)
 
-target_link_libraries(osrm-example ${LibOSRM_LIBRARIES} ${LibOSRM_DEPENDENT_LIBRARIES})
+target_link_libraries(osrm-example ${LibOSRM_LIBRARIES} ${LibOSRM_DEPENDENT_LIBRARIES} ${UtilGlob})
```

Comment:
Will check, don't remember exactly. I recall this: `osrm-example` uses `io_config` which uses `util/log`.